drivers/storage/portworx/util/util.go

```diff
@@ -272,6 +272,7 @@ const (
 	OpenshiftPrometheusSupportedVersion = "4.12"
 	Openshift_4_15_Version              = "4.15"
 	Openshift_4_16_version              = "4.16"
+	OpenshiftPxSupportedVersion         = "4.13"
```

Comment:
Where are we using OpenshiftPxSupportedVersion ?

Reply:
This was used to check supported ocp version while defaulting to dmthin. But dmthin is not default now, will remove it.